doc/integrator/querier.rst

```diff
@@ -25,16 +25,16 @@ The web service configuration is done in the ``vars_<project>.yaml`` file:
                         separator: ","
 
 The ``dbsession: "<session name>"`` in the ``enum`` is just a shortcut
-if almost of the session are similar in the layers. If the dbsession isn't
-defined we use the main DB session.
+if almost of the session are similar in the layers. If ``dbsession`` is not
```

Comment:
I guess that this means that one may specify the dbsession (but with what value??) as the default session and override it for each layer if needed.